src/handlers/response-interceptor.ts

```diff
@@ -12,6 +12,19 @@ type Interceptor = (
   res: http.ServerResponse
 ) => Promise<Buffer | string>;
 
+const TrailerDisallowHeaders: string[] = [
```

Comment:
Could you add a link to verify the completeness of these headers?

Reply:
https://developer.mozilla.org/docs/Web/HTTP/Headers/Trailer